### PR TITLE
Q&M: Fix broken links identified by the Broken links report - 2

### PR DESCRIPTION
**Pull request by Albertyang0**

### CODE REVIEW

**Comment by Albertyang0** [comment]
@dotnet-policy-service agree company="Microsoft"

**Comment by Albertyang0** [comment]
@cmastr, @Rick-Anderson, @wadepickett, @tdykstra, I've fixed some broken links, please verify. Thanks.

File: aspnet/mvc/overview/older-versions/using-the-html5-and-jquery-ui-datepicker-popup-calendar-with-aspnet-mvc/using-the-html5-and-jquery-ui-datepicker-popup-calendar-with-aspnet-mvc-part-1.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ by [Rick Anderson](https://twitter.com/RickAndMSFT)
...
 
-Before you start, make sure you've installed the prerequisites listed below. You can install all of them by clicking the following link: [Web Platform Installer](https://www.microsoft.com/web/gallery/install.aspx?appid=VWD2010SP1Pack). Alternatively, you can individually install the required software using the following links:
+Before you start, make sure you've installed the prerequisites listed below. You can install all of them by clicking the following link: [Web Platform Installer](https://my.visualstudio.com/Downloads?q=visual%20studio%202010%20service%20pack%201). Alternatively, you can individually install the required software using the following links:
 
-- [Visual Studio Web Developer Express SP1 prerequisites](https://www.microsoft.com/web/gallery/install.aspx?appid=VWD2010SP1Pack)
+- [Visual Studio Web Developer Express SP1 prerequisites](https://my.visualstudio.com/Downloads?q=visual%20studio%202010%20service%20pack%201)
```
Comment (wadepickett) [inline aspnet/mvc/overview/older-versions/using-the-html5-and-jquery-ui-datepicker-popup-calendar-with-aspnet-mvc/using-the-html5-and-jquery-ui-datepicker-popup-calendar-with-aspnet-mvc-part-1.md:22]:
```suggestion
- [Visual Studio Web Developer Express SP1 prerequisites](https://my.visualstudio.com/Downloads?q=Visual%20Studio%20Web%20Developer%20Express)
```
SP1 is no longer available anywhere, but the 6/2010 release is available so I will change this to point to that.

**Review by wadepickett — APPROVED** [review]
@Albertyang0, great work finding the best possible link replacements on what is very old content!

**Pull request merged**